user/how-to-guides/how-to-organize-your-qubes.md

```diff
@@ -0,0 +1,451 @@
+them organized by coming up with a naming scheme, such as:
+
+```
+client1-code
+client1-build
+client1-test
+client1-prod
+client2-code
+client2-build
+client2-test 
+client2-prod
+...
+```
+
+This helps her keep groups of qubes organized in a set. Some of her qubes are
+based on [Debian templates](/doc/templates/debian/), while others are based on
+[Fedora templates](/doc/templates/fedora/). The reason for this is that some
+software packages are more readily available in one distribution as opposed to
+the other. Alice's setup looks like this:
+
+- Several qubes for writing code. Here's where she runs her IDE, commits code,
+  and signs her commits. These qubes are based on different templates depending
+  on which tools and which development environment she needs. In general, Alice
+  likes to have a separate qube of this type for each client or each project.
+  This allows her to keep everything organized and avoid accidentally mixing up
+  any access credentials or client code, which could be disastrous. This also
+  allows her to truthfully tell her clients that their code is always securely
+  isolated from all her other clients. She likes to use the [Qubes
+  firewall](/doc/firewall/) to restrict these qubes' network access to only the
+  code repositories she needs in that qube in order to avoid accidentally
+  interacting with anything else on her local network or on the internet. Alice
+  also has some qubes of this type for personal programming projects that she
+  works on just for fun when she has "free time" (whatever that is).
+
+- Several qubes for building and testing. Again, Alice usually likes to have
+  one of these for each project in order to keep things organized. Here's where
+  she pulls any dependencies she needs, compiles her code, runs her build
+  toolchain, and tests her deliverables. In some cases, she finds it useful to
+  use [standalones](/doc/standalones-and-hvms/) for these so that it's easier
+  to quickly [install different pieces of
+  software](/doc/how-to-install-software/) without having to juggle rebooting
+  both the template and an app qube. She also sometimes finds it necessary (or
+  just convenient) to make edits to config files in the root filesystem, and
+  she'd rather not have to worry about losing those changes during an app qube
+  reboot. She knows that she could use [bind-dirs](/doc/bind-dirs/) to make
+  those changes persistent, but sometimes she doesn't want to get bogged down
+  doing with all that and figures it wouldn't be worth it just for this one
+  qube. She's secretly glad that Qubes OS doesn't judge her this and just gives
+  her the freedom to do things however she likes while keeping everything
+  securely compartmentalized. At times like these, she takes comfort in knowing
+  that things can be messy and disorganized *within* a qube while her overall
+  digital life remains well-organized.
+
+- Several email qubes. Since Alice is a serious programmer, she likes to use a
```

Comment:
I'm mildly opposed about "serious programmer" being equivalent to "uses command line client" :) GUI tools are not inherently inferior or less serious (and I considered myself a serious programmer even if I do use Thunderbird). Maybe just "Alive prefers flexibility and configurability of command-line tools"?

Reply:
Ah, that was meant to be tongue-in-cheek, but perhaps it didn't land. >.<

I had thought about, "Since Alice fancies herself a serious programmer..." or "Since Alice likes to think of herself as a serious programmer..." but those sounded mildly insulting to programmers who happen to like Mutt. :)

Reply:
Haha, didn't land for me at least - but I may have the "people don't consider making UI real programming and it makes me mad" bias :D